### ISSUE TITLE
Directives with falsy values are added to csp string

### ISSUE THREAD

**Issue by inv8der** [helmetjs/content-security-policy-builder#3]
Ran into this issue using `helmet-csp`. While running our FE server locally (not using https), all requests to resources served by the FE server were failing.

```
var cspBuilder = require("content-security-policy-builder");

cspBuilder({
    directives: {
        childSrc: [ '*.zendesk.com', '*.cloud9charts.com', '\'self\'' ],
        upgradeInsecureRequests: false 
    }
})
```

**Result**
```
child-src *.zendesk.com *.cloud9charts.com 'self'; upgrade-insecure-requests
```

**Comment by EvanHahn** [comment]
This is intentional. Setting a directive to `false` will include it with no values. This is in contrast to [helmet-csp](https://github.com/helmetjs/csp), where `false` will omit the key.

I realize that this is inconsistent—two Helmet modules for CSP behave differently in this case.

Does that help?

**Comment by graingert** [comment]
@EvanHahn yeah that's very confusing, can you make them work the same?

**Comment by EvanHahn** [comment]
I agree that it's confusing. Would you be interested in making a pull request?

**Comment by EvanHahn** [comment]
Anyone interested in making a pull request?

**Comment by EvanHahn** [comment]
Bumping this. Is anyone interested in making a pull request? If not, this issue will stay open for awhile.

**Comment by EvanHahn** [comment]
This has been fixed in `content-security-policy-builder@2.0.0`.

**Comment by graingert** [comment]
great stuff, thanks @EvanHahn
